api/dao/containerutil.py

```diff
@@ -37,6 +37,32 @@ def add_id_to_subject(subject, pid):
         subject['_id'] = bson.ObjectId()
     return subject
 
+def get_stats(cont, cont_type):
```

Comment:
I would like to move this to `containerstorage.py`, as well as `add_id_to_subject()` above, but `session-templates` contains the rework. 

This will most likely merge first so I'll make the move in the rebase of `session-templates` when this lands.